_commands/basics/kill.md

```diff
@@ -3,12 +3,13 @@
 
 kill
 --
-TODO: Add documentation for this command by submitting a pull request.
+The kill command is used on Linux systems to terminate process by its process id without having to reboot the system.
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+The syntax is kill [signal or option] PID(s)
+$kill 27410         Means killing a current running process with process id 27410
```

Comment:
Could you use a comment or some other change of format so it doesn't seem like `Means killing a current running process with process id 27410` is a part of the command?